src/test/java/com/wikia/webdriver/testcases/mercurytests/PortableInfoboxTests.java

```diff
@@ -132,4 +133,40 @@ public void MercuryPortableInfoboxTest_006_DifferentClickTargets() {
         .clickGalleryButton(0)
         .isInfoboxExpanded();
   }
+
+  @Test(groups = "MercuryPortableInfoboxTest_007")
+  public void MercuryPortableInfoboxTest_007_HeroImageCropping() {
+    PortableInfoboxObject infoboxTallImage = new PortableInfoboxObject(driver);
+    infoboxTallImage.openArticleOnWikiByNameWithCbAndNoAds(wikiURL, MercurySubpages.INFOBOX_1);
+    infoboxTallImage.isHeroImageSquare();
+
+    PortableInfoboxObject infoboxWideImage = new PortableInfoboxObject(driver);
+    infoboxWideImage.openArticleOnWikiByNameWithCbAndNoAds(wikiURL, MercurySubpages.INFOBOX_4);
+    infoboxWideImage.isNotHeroImageSquare();
+  }
+
+  @Test(groups = "MercuryPortableInfoboxTest_008")
+  public void MercuryPortableInfoboxTest_008_ImageCollection() {
+    PortableInfoboxObject info = new PortableInfoboxObject(driver);
+    info.openArticleOnWikiByNameWithCbAndNoAds(wikiURL, MercurySubpages.INFOBOX_5);
+
+    info.isImageInCollectionVisible()
+        .clickNextImageArrow()
+        .isImageInCollectionVisible()
+        .clickNextImageArrow()
+        .isImageInCollectionVisible();
+  }
+
+  @Test(groups = "MercuryPortableInfoboxTest_009")
+  public void MercuryPortableInfoboxTest_009_HeadersInTOC() {
+    PortableInfoboxObject info = new PortableInfoboxObject(driver);
+    info.openArticleOnWikiByNameWithCbAndNoAds(wikiURL, MercurySubpages.INFOBOX_2);
+
+    String headerName = info.getHeaderName(0);
+
+    TableOfContentPageObject toc = new TableOfContentPageObject(driver);
```

Comment:
move this declaration under `PortableInfoboxObject info = new PortableInfoboxObject(driver);`